src/main/java/org/broadinstitute/hellbender/utils/pairhmm/VectorLoglessPairHMM.java

```diff
@@ -101,12 +101,16 @@ public VectorLoglessPairHMM() throws UserException.HardwareFeatureException {
                     logger.info("libVectorLoglessPairHMM found in JVM library path");
                 } catch (UnsatisfiedLinkError e) {
```

Comment:
We could do it enterprise style and provide an `AVXAvailableProviderFactory` and then mock that...